### PR TITLE
W-14856539 acb commands duke

### PR DESCRIPTION
**Pull request by dukesphere**
you can find list of all commands from Preferences: Open Keyboard Shortcuts and searching for MuleSoft

### CODE REVIEW

File: modules/ROOT/pages/acb-commands.adoc [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+The IDE provides a set of commands through the Command Palette. 
+
+== MuleSoft Commands
+
+// recommended change-> Add Fragment Dependency from Exchange
+* `MuleSoft: Add Fragment dependency from Exchange`
++
+Add an API spec fragment from Exchange to the project directory so that you can include the fragment in your spec. For a procedure that uses this command, see xref:des-create-api-fragments.adoc#add-fragment-to-project[Add a Fragment from Exchange to Your Project].
+
+//not available
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+== MuleSoft Commands
+
+// recommended change-> Add Fragment Dependency from Exchange
+* `MuleSoft: Add Fragment dependency from Exchange`
++
+Add an API spec fragment from Exchange to the project directory so that you can include the fragment in your spec. For a procedure that uses this command, see xref:des-create-api-fragments.adoc#add-fragment-to-project[Add a Fragment from Exchange to Your Project].
+
+* `MuleSoft: API Console`
++
+Open the API Console for a design project. See xref:des-create-api-specs.adoc[].
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
Open the API Console from a design project. See xref:des-create-api-specs.adoc[].
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+* `MuleSoft: Deploy to CloudHub`
++
+Deploy an integration or implementation to CloudHub. See xref:int-deploy-mule-apps.adoc#deploy-an-app-to-cloudhub[Deploy an App to CloudHub].
+
+* `MuleSoft: Design an API`
++
+Open a form for starting an API design project. You can also start an API design project from the UI. See xref:des-create-api-specs.adoc[].
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
Start an API design project. You can also start an API design project from the UI. See xref:des-create-api-specs.adoc[].
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+* `MuleSoft: Develop an Integration`
++
+Open a form for starting an integration project. You can also start an integration project from the UI. See xref:int-create-integrations.adoc[].
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
Start an integration project. You can also start an integration project from the UI. See xref:int-create-integrations.adoc[].
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
++
+Publish an API spec or fragment from the IDE to Anypoint Exchange. See xref:des-publish-api-spec-to-exchange.adoc##publish-spec[Publish an API spec to Exchange].
+
+// recommended change-> "this local" -> "This Local"
+* `MuleSoft: Re-scaffold this local API`
++
+After scaffolding an API design project from the IDE into an interface within an implementation project and making subsequent changes to the _design_ project, use this command from the design project to re-scaffold your changes into your interface. See xref:imp-implement-local-apis.adoc#rescaffold-api-spec[Re-Scaffold the API Spec].
+
+//
+// not clear what this one does
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
// not clear what this one does when you try it out
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+//* `MuleSoft: Refresh DataSense Results`
+//
+// not clear what this one does
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
// not clear what this one does when you try it out
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+* `MuleSoft: Develop an Integration`
++
+Start an integration project. You can also start an integration project from the UI. See xref:int-create-integrations.adoc[].
+
+* `MuleSoft: Export Mule Application Sources`
++
+Export a sharable JAR file that contains a integration or implementation project. Both types are projects are Mule applications. See xref:int-export-mule-project.adoc#sharable[Export to a Shareable JAR File]. 
+
+//not available
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+// recommended change-> Log In to Anypoint Platform
+* `MuleSoft: Login to Anypoint Platform`
++
+Log in to Anypoint Platform from the IDE. See xref:start-acb.adoc#login-ide[Log in from the IDE through the Command Palette].
+
+* `MuleSoft: New Mule Configuration File`
++
+Add a configuration XML file to your integration or implementation project. See xref:int-create-integrations.adoc#create-new-config[Create an Additional Configuration XML File].
+
+//TODO: See PR: https://github.com/mulesoft/docs-code-builder/pull/256/files
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
//TODO: See PR: https://github.com/mulesoft/docs-code-builder/pull/256/files (settings doc, questions remain)
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+// "ACB" is not used, could be a legal naming issue
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
// "ACB" is not used as name for product, ok to use?
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+// recommended change-> Log In to Anypoint Platform
+* `MuleSoft: Login to Anypoint Platform`
++
+Log in to Anypoint Platform from the IDE. See xref:start-acb.adoc#login-ide[Log in from the IDE through the Command Palette].
+
+* `MuleSoft: New Mule Configuration File`
++
+Add a configuration XML file to your integration or implementation project. See xref:int-create-integrations.adoc#create-new-config[Create an Additional Configuration XML File].
+
+//TODO: See PR: https://github.com/mulesoft/docs-code-builder/pull/256/files (settings doc, questions remain)
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
//TODO: supplemental info in PR: https://github.com/mulesoft/docs-code-builder/pull/256/files (settings doc, questions remain)
```

File: modules/ROOT/pages/acb-commands.adoc [hunk 11]
```diff
@@ -0,0 +1,164 @@
+= Commands in Anypoint Code Builder
+
+The IDE provides a set of commands through the Command Palette. 
+
+== MuleSoft Commands
+
+// recommended change-> Add Fragment Dependency from Exchange
+* `MuleSoft: Add Fragment dependency from Exchange`
++
+Add an API spec fragment from Exchange to the project directory so that you can include the fragment in your spec. For a procedure that uses this command, see xref:des-create-api-fragments.adoc#add-fragment-to-project[Add a Fragment from Exchange to Your Project].
```
Comment (martin-scovotti) [inline modules/ROOT/pages/acb-commands.adoc:10]: Are we using _spec_ rather than _Specification_ in our documentation?

File: modules/ROOT/pages/acb-commands.adoc [hunk 12]
```diff
@@ -0,0 +1,164 @@
...
++
+Export a Mule project with its compiled code and all dependencies into a distributable JAR file. See xref:int-export-mule-project.adoc#deployable[Export to a Deployable JAR File].
+
+* `MuleSoft: Export Troubleshooting Information`
++
+Export a compressed file that contains your project’s workspace folders, logs, and information about environment variables, CPU, operating system, and system properties. See xref:troubleshooting.adoc#export-troubleshooting-info[Export Troubleshooting Information].
+
+* `MuleSoft: Implement an API Specification`
++
+Import a supported API spec from Anypoint Exchange, and scaffold the spec into an implementation project in the IDE. See xref:imp-implement-apis.adoc#scaffold-new-integration[Scaffold an API and Create an Integration Project].
```
Comment (martin-scovotti) [inline modules/ROOT/pages/acb-commands.adoc:43]: _spec_ usage, same as above

**Review by martin-scovotti — APPROVED** [review]
LGTM

File: modules/ROOT/pages/acb-commands.adoc (hunk 11 above)
Comment (dukesphere) [reply modules/ROOT/pages/acb-commands.adoc:10]: Yeah, we started using "spec" ¯\_(ツ)_/¯

File: modules/ROOT/pages/acb-commands.adoc [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
++
+Start an API design project. You can also start an API design project from the UI. See xref:des-create-api-specs.adoc[].
+
+* `MuleSoft: Develop an Integration`
++
+Start an integration project. You can also start an integration project from the UI. See xref:int-create-integrations.adoc[].
+
+* `MuleSoft: Export Mule Application Sources`
++
+Export a sharable JAR file that contains a integration or implementation project. Both types are projects are Mule applications. See xref:int-export-mule-project.adoc#sharable[Export to a Shareable JAR File].
```
Comment (chris-campbell9) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
Export a shareable JAR file that contains an integration or implementation project. Both types are projects that are Mule applications. See xref:int-export-mule-project.adoc#sharable[Export to a Shareable JAR File]. 
```
I notice the anchor in the `int-export-mule-project.adoc` xref contains the same typo. Does the heading for that section also contain the typo?

File: modules/ROOT/pages/acb-commands.adoc [hunk 14]
```diff
@@ -0,0 +1,164 @@
...
++
+Start an API design project. You can also start an API design project from the UI. See xref:des-create-api-specs.adoc[].
+
+* `MuleSoft: Develop an Integration`
++
+Start an integration project. You can also start an integration project from the UI. See xref:int-create-integrations.adoc[].
+
+* `MuleSoft: Export Mule Application Sources`
++
+Export a shareable JAR file that contains an integration or implementation project. Both types are projects that are Mule applications. See xref:int-export-mule-project.adoc#sharable[Export to a Shareable JAR File].
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-commands.adoc]: looks like I misspell it everywhere. 😆 
fixing

Comment (dukesphere) [reply modules/ROOT/pages/acb-commands.adoc]: fixed everywhere Thx for catching that!

File: modules/ROOT/pages/acb-commands.adoc [hunk 15]
```diff
@@ -0,0 +1,164 @@
...
+Scaffold an API spec project located within the IDE into an implementation project. See xref:imp-implement-local-apis.adoc[Scaffold an API Spec into a New Implementation Project]. 
+
+* `MuleSoft: Import a Mule Project`
++
+Import a JAR file for a Mule project and load the unpacked project into a workspace in the IDE. A Mule project is an integration or implementation project. See xref:int-import-mule-project.adoc#desktop-project-package[Import a Shareable Project JAR File into the Desktop IDE] or xref:int-import-mule-project.adoc#web-project-package[Import a Shareable Project JAR File into the Cloud IDE].
+
+//TODO: "runnable Mule application" ok? 
+* `MuleSoft: Import a Project`
++
+Import either a design, integration, or implementation project. Import API specifications or fragments from Anypoint Exchange, or JAR file for a runnable Mule application (integration or implementation) in your IDE directory. See xref:int-import-mule-project.adoc[].
```
Comment (chris-campbell9) [inline modules/ROOT/pages/acb-commands.adoc]:
```suggestion
Import a design, integration, or implementation project. Import API specs or fragments from Anypoint Exchange, or a JAR file for a runnable Mule application (integration or implementation) in your IDE directory. See xref:int-import-mule-project.adoc[].
```
IIRC in most cases "either" refers to two items, but occasionally it can refer to more than two. Total nit-picky suggestion! :) Please feel free to disregard!

Regarding "specifications", elsewhere we're just saying "spec". Should it be consistent?

File: modules/ROOT/pages/acb-commands.adoc [hunk 16]
```diff
@@ -0,0 +1,164 @@
...
+//
+
+* `MuleSoft: Publish API Specification to Exchange`
++
+Publish an API spec or fragment from the IDE to Anypoint Exchange. See xref:des-publish-api-spec-to-exchange.adoc##publish-spec[Publish an API spec to Exchange].
+
+// recommended change-> "this local" -> "This Local"
+* `MuleSoft: Re-scaffold this local API`
++
+After scaffolding an API design project from the IDE into an interface within an implementation project and making subsequent changes to the _design_ project, use this command from the design project to re-scaffold your changes into your interface. See xref:imp-implement-local-apis.adoc#rescaffold-api-spec[Re-Scaffold the API Spec].
```
Comment (chris-campbell9) [inline modules/ROOT/pages/acb-commands.adoc:101]:
```suggestion
After scaffolding an API design project from the IDE into an implementation project interface and making subsequent changes to the _design_ project, use this command from the design project to re-scaffold your changes into your interface. See xref:imp-implement-local-apis.adoc#rescaffold-api-spec[Re-Scaffold the API Spec].
```
At first glance, that first sentence seems a bit wordy? Can it be shortened and simplified? I'm not sure if my suggestion helps. After multiple re-reads it made sense, so maybe not a big deal.

Comment (dukesphere) [reply modules/ROOT/pages/acb-commands.adoc:101]: I want them to emphasize that they are scaffolding the spec into an interface that becomes the implementation project

**Comment by chris-campbell9** [comment]
@dukesphere Hey, I’m not seeing the new `acb-commands.adoc` file in the nav page. It’s only linked from `acb-reference.adoc`. Should it be in the nav? Is it considered an “orphan” if it’s not in the nav?

File: modules/ROOT/pages/acb-commands.adoc [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+//ISSUE: doesn't show up in the IDE
+//* `MuleSoft: Show Mule Graphical Mode`
+//
+
+//recommended change-> Show Project Dependencies
+* `MuleSoft: Show Project dependencies`
++
+Open the *Project Dependencies* area of the Explorer view for a project. All types of projects can have dependencies. Integration and implementation projects list Java packages in the area. API specification projects list any API fragment dependencies in the area, for example: 
++
+images::des-api-frag-dependencies.png["Example of API fragments listed in the Project Dependencies area"]
```
Comment (chris-campbell9) [inline modules/ROOT/pages/acb-commands.adoc]: This image isn't rendering when I build locally.

Comment (dukesphere) [reply modules/ROOT/pages/acb-commands.adoc]: hmm, lemme check

**Review by chris-campbell9 — APPROVED** [review]

File: modules/ROOT/pages/acb-commands.adoc (hunk 17 above)
Comment (dukesphere) [reply modules/ROOT/pages/acb-commands.adoc]: fixed! thx for noticing that!!
